wtxmgr/tx_test.go

```diff
@@ -1522,6 +1522,80 @@ func TestInsertMempoolTxAlreadyConfirmed(t *testing.T) {
 	checkStore()
 }
 
+func TestInsertMempoolTxAfterSpentOutput(t *testing.T) {
```

Comment:
One last thing: would be nice to provide a small description for the test as a comment like the other tests within this file.

Reply:
Done.